plugins/modules/ssh_config.py

```diff
@@ -50,8 +50,11 @@
   host:
     description:
       - The endpoint this configuration is valid for.
-      - Can be an actual address on the internet or an alias that will
-        connect to the value of O(hostname).
+      - Can be an actual address on the internet, an alias that will
+        connect to the value of O(hostname), or V(*) to match all hosts.
+      - If host is V(*) it is best to place it as the last host since ssh
+        goes with the first matching option. This will make sure that any
+        other host options take precedence over the V(*) host.
```

Comment:
The docs should also say how to place that entry at the bottom.

Reply:
This still needs to be covered somehow. Right now the module adds new entries at the bottom, even if `*` comes further up.